eng/Versions.props

```diff
@@ -3,15 +3,15 @@
     <!-- Current previous .NET SDK major version's stable release of MAUI packages -->
     <MicrosoftMauiPreviousDotNetReleasedVersion>6.0.548</MicrosoftMauiPreviousDotNetReleasedVersion>
     <!-- dotnet/installer -->
-    <MicrosoftDotnetSdkInternalPackageVersion>7.0.100-rtm.22519.39</MicrosoftDotnetSdkInternalPackageVersion>
+    <MicrosoftDotnetSdkInternalPackageVersion>7.0.101-servicing.22571.1</MicrosoftDotnetSdkInternalPackageVersion>
```

Comment:
can this be the public 7.0.101 now?

Reply:
The reason you'd use a nightly build like `7.0.101-servicing.22571.1`, is so you can test changes before they ship.

Reply:
Right but for example this failed to build and get merged before the servicing coming out.. we should have been able to test this earlier or get more people trying to fix the issues.
 
 also if I change to 7.0.101 it doesn't install, can't find it ..

Reply:
`./dotnet-install.sh --version 7.0.101` won't work for stable builds. You'd have to use `./dotnet-install.sh --channel 7.0.1xx --quality stable`.